int32 cnt
float32[200] x_arr
float32[200] y_arr
